### ISSUE TITLE
Crash if unused emit native

### ISSUE THREAD

**Issue by samphunter** [pawn-lang/compiler#412]
### Issue description:

Using `emit sysreq.c native;` without using the native elsewhere in the code will crash compiler.

### Minimal complete verifiable example (MCVE):

```
native print(const str[]);

main() {
    new str[12];
    emit {
        push.adr str
        push.c 4
        sysreq.c print
        stack 8
    }
}
```

### Workspace Information:

* Compiler version: 3.10.9
* Command line arguments provided (or sampctl version): -;+ -)+
* Operating System: Windows 8.1

**Comment by Daniel-Cortez** [comment]
I think I fixed this bug in a local repo back in December, as part of a batch of fixes for the `__emit` operator. Now I'm just waiting until #393 is merged before making my own PR, in order to avoid any further conflicts.

**Comment by Daniel-Cortez** [comment]
Fixed in #421.
